content/reference/drops/time_drop.md

```diff
@@ -0,0 +1,14 @@
+# Time Drop
+
+Return attributes with time. You have ability to manage time format.
```

Comment:
This is a weird drop, named `Time` and it returns dates only
@ZenCocoon Maybe rename the drop? Is it even used?